flow/designs/asap7/mock-array-big/config.py

```diff
@@ -1,27 +1,44 @@
-# see flow/platforms/asap7/openRoad/pdn/grid_strategy-M2-M5-M7.tcl
 import os
 
+# number of Elements in row and column, user can set via environment variable
 rows = int(os.environ.get("MOCK_ARRAY_HEIGHT", "8"))
 cols = int(os.environ.get("MOCK_ARRAY_WIDTH", "8"))
+
+# must be 8x8
+if (rows * cols) < 64:
+  print("ERROR: number of elements row and columns must add up to 64.")
```

Comment:
I would like to avoid this restriction, can it simply be removed?

We are investigating other layouts, like a really large element with 1x1 array or a large 16x8 array.